### ISSUE TITLE
Suggest classifying Static Weapon groups as Special Forces 

### ISSUE THREAD

**Issue by NZF-JDWang** [tupolov/cfp#87]
As per [this ](http://alivemod.com/forum/4574-are-there-any-real-differences-in-how-the-opcom-uses-groups/0#p26376) thread, the ALiVE OPCOM doesn't use the ORBAT groups differently, it just weights the spawning.

It's been my opinion that the number of Mortar Teams/Heavy Weapon/Sniper teams which spawn is too high compared to regular infantry, making them all too common on the battlefield.

Classifying them as Special Forces in the ORBAT will reduce the spawn rate of these special groups and bring them more into line with what you would expect to see on the battlefield.

Using the Taliban Insurgents as an example, there are currently 10 groups, 4 of which are heavy weapons groups, 1 sniper team and 1 demo team. This means there's only a 40% chance that when a group spawns it's going to be a plain old infantry group.

For the mission I'm working on I have copied the 4 man team a couple of times and the 6 man team once, I've moved the sniper team, and heavy weapons teams into the Special Forces category and even moved the mortar team into "Artillery" (which I know wouldn't work for every faction)

These changes mean I'm now much more likely to run into a small group of insurgents, heavy weapons teams are still around but vastly more rare, and with the mortar team classified as artillery I can specify in the placement modules that I want 2 teams specifically, without having to manually place them on the map. (again I realize this is special case with the Taliban not having any but I use it as an example of how the group classifications could be used better) 

TL;DR Classifying Static Weapon/Sniper teams as Special Forces will not only make them more rare, but also give mission makers the option in the placement modules to specify the number of HW teams separate from regular infantry.

**Comment by AUTigerGrad** [comment]
This is a good idea and has been implemented.
